test/development/acceptance/ReactRefreshLogBox-app-doc.test.ts

```diff
@@ -96,41 +96,41 @@ describe.each(['default', 'turbo'])(
       const source = next.normalizeTestDirContent(content)
       if (process.env.TURBOPACK) {
         expect(source).toMatchInlineSnapshot(`
-        "./pages/_app.js:2:11
-        Parsing ecmascript source code failed
-          1 | function MyApp({ Component, pageProps }) {
-        > 2 |   return <<Component {...pageProps} />;
-            |            ^^^^^^^^^
-          3 | }
-          4 | export default MyApp
-
-        Expression expected"
-      `)
+          "./pages/_app.js:2:11
+          Parsing ecmascript source code failed
+            1 | function MyApp({ Component, pageProps }) {
+          > 2 |   return <<Component {...pageProps} />;
+              |           ^
+            3 | }
+            4 | export default MyApp
+
+          Expression expected"
+        `)
       } else {
         expect(source).toMatchInlineSnapshot(`
-        "./pages/_app.js
-        Error: 
-          x Expression expected
-           ,-[TEST_DIR/pages/_app.js:1:1]
-         1 | function MyApp({ Component, pageProps }) {
-         2 |   return <<Component {...pageProps} />;
-           :           ^
-         3 | }
-         4 | export default MyApp
-           \`----
-
-          x Expression expected
-           ,-[TEST_DIR/pages/_app.js:1:1]
-         1 | function MyApp({ Component, pageProps }) {
-         2 |   return <<Component {...pageProps} />;
-           :            ^^^^^^^^^
-         3 | }
-         4 | export default MyApp
-           \`----
-
-        Caused by:
-            Syntax Error"
-      `)
+                  "./pages/_app.js
```

Comment:
why is this indented so far now? 🙈

Reply:
I rerecorded with Jest and it did this 🙈